packages/cspell-glob/package.json

```diff
@@ -40,9 +40,11 @@
     "node": ">=12.0.0"
   },
   "dependencies": {
+    "@types/node": "^14.14.6",
```

Comment:
Maybe it is best to have this in `devDependencies`. What node types are being exported? Maybe that is the issue.

Reply:
Believe I got that from https://github.com/streetsidesoftware/cspell/blob/c8b115db4916f4d31ebfc10972437030e71fac3f/packages/cspell-glob/src/GlobMatcher.ts#L6

Reply:
Fixed with #554